test/Interop/Cxx/class/nested-records-module-interface.swift

```diff
@@ -82,7 +82,7 @@
 // CHECK:   struct ForwardDeclaredFriend {
 // CHECK:     init()
 // CHECK:   }
-// CHECK:   static func takesFriend(_ b: NestedDeclIsAFirstForwardDeclaration.ForwardDeclaredFriend)
```

Comment:
This seems odd. Any ideas why this changed?

Reply:
Previously we went through all the decls in a namespace, and emitted them, and now we request a canonical decl for each of the decls in a namespace, to make sure we don't emit the same decl multiple times if it has redecls inside different redecls of the namespace.
Different redecls of this function have different parameter names, so I had to update this test.